bot/persistence/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Module containing persistence logic and accessors for Database. The only needed class to initialize and
+access the database is the DatabaseConnector.
```

Comment:
Docstring format isn't 100% accurate. Take a look at this:
[https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)

Reply:
I don't believe I am thaaat far away from the recommended google style, as it allows for a lot of freedom, however I will still adapt it as good as possible